### PR TITLE
feat(issue-20): implement approve and change request

### PR DESCRIPTION
**Pull request by jskurt**

### CODE REVIEW

File: src/modules/github/components/PullReviewForm.tsx [hunk 1]
```diff
@@ -93,7 +125,7 @@ export const PullReviewForm: FC<PullReviewFormProps> = ({
         className="w-96 h-40"
         value={commentInput}
         onChange={handleCommentChange}
-        disabled={!hasChecked}
+        disabled={!hasChecked || commentInput.length <= 0}
```
Comment (kafelix496) [inline src/modules/github/components/PullReviewForm.tsx:128]: You can just do `length === 0`. length can't be smaller than 0

File: src/modules/github/components/PullReviewForm.tsx [hunk 2]
```diff
@@ -85,6 +85,38 @@ export const PullReviewForm: FC<PullReviewFormProps> = ({
     })
   }
 
+  const handleApproveClick = () => {
+    console.log('approve click')
+
+    const checkedPullsInfo = getCheckedPullsInfo(repoHasCheckArray)
```
Comment (kafelix496) [inline src/modules/github/components/PullReviewForm.tsx:91]: this is the reason i separate `getCheckedPullsInfo` function so you can use it for "comment", "approve", and "request change".

Comment (jskurt) [reply src/modules/github/components/PullReviewForm.tsx:91]: uhm, okay!

**Comment by jskurt** [comment]
I don't know exactly that Is it really done?

File: src/modules/github/components/PullReviewForm.tsx [hunk 3]
```diff
@@ -85,6 +85,38 @@ export const PullReviewForm: FC<PullReviewFormProps> = ({
     })
   }
 
+  const handleApproveClick = () => {
+    console.log('approve click')
```
Comment (kafelix496) [inline src/modules/github/components/PullReviewForm.tsx:89]: remove console

**Review by kafelix496 — APPROVED** [review]

**Pull request merged**